src/test/java/org/jitsi/meet/test/MuteTest.java

```diff
@@ -159,29 +159,25 @@ public void ownerMutesParticipantAndCheck()
 
         WebElement elem = owner.findElement(By.xpath(
             TestUtils.getXPathStringForClassName("//span", "remotevideomenu")
-            + "/i[@class='icon-thumb-menu']"));
+            + "//i[@class='icon-thumb-menu']"));
```

Comment:
But then again why not? It is better for readability. I'll just make the change now.